dev_cmds/integration-tests/single-package-json-example.ts

```diff
@@ -64,9 +63,32 @@ export function createSinglePackageJsonExampleTestGroup(): TestGroup {
     }
   };
 
-  const runFailsWithErrorCmd: TestFunction = async (containerName: string) => {
-    const exampleCmdCommandLine = `npx devcmd fails_with_error`;
+  const runExampleCmdWithExtension: TestFunction = async (containerName: string) => {
+    const { stdout, stderr } = await execToString({
```

Comment:
IIRC there are several calls of ```execToString``` that all share the following lines

````typescript
command: DOCKER_COMMAND,
args: [
  "exec",
  containerName,
  "sh",
  "-c",
```

Do you think it might be a good idea to increase the readability of the integration tests by adding a util function like ```export function runInTestDocker(containerName: string, command: string)``` ?

Reply:
good point! I had thought about this before, but now seems to be a good time. Since this will touch several other test cases whose content isn't related to this PR, I'll create a separate PR for that though :)